web/src/about_zulip.ts

```diff
@@ -17,11 +17,15 @@ export function launch(): void {
         },
     });
 
-    const clipboard = new ClipboardJS("#about-zulip .fa-copy");
-
-    clipboard.on("success", () => {
+    const zulip_version_clipboard = new ClipboardJS("#about-zulip .fa-copy.zulip-version");
+    zulip_version_clipboard.on("success", () => {
         show_copied_confirmation($("#about-zulip .fa-copy")[0]);
     });
+
+    const zulip_merge_base_clipboard = new ClipboardJS("#about-zulip .fa-copy.zulip-merge-base");
+    zulip_merge_base_clipboard.on("success", () => {
+        show_copied_confirmation($("#about-zulip .fa-copy")[1]);
```

Comment:
Since we now have different classes to separate them, why don't we use the complete selector to avoid ambiguity?
```js
show_copied_confirmation($("#about-zulip .fa-copy.zulip-merge-base"));
```

Reply:
Fixed in latest commit.